packages/office-ui-fabric-react/src/components/DocumentCard/DocumentCard.base.tsx

```diff
@@ -32,7 +41,7 @@ export class DocumentCard extends BaseComponent<IDocumentCardProps, any> impleme
     }
 
     // if this element is actionable it should have an aria role
-    const role = this.props.role || (actionable ? (onClick ? 'button' : 'link') : undefined);
```

Comment:
Do we need to deprecate this prop now that it's unused?

Reply:
Which prop is deprecated?